tests/testthat/test_OS_key.R

```diff
@@ -0,0 +1,20 @@
+context("test api key")
+
+test_that("multiplication works", {
+  expect_equal(2 * 2, 4)
+})
+
+
+
+test_that("set_api_key()", {
+						expect_error(set_os_key())
+#						expect_false(set_os_key(""))
+						expect_true(set_os_key(Sys.getenv("API_KEY")))
```

Comment:
This won't work unless there is an environment variable called `API_KEY`. If one to run these tests on their own it'll fail. I think it might be better to hard code some random character value and check it something like. 

```
test_key <- "a-test-value"
set_os_key(test_key)

expect_equal(test_key, get_os_key())
```